packages/valory/skills/price_estimation_abci/payloads.py

```diff
@@ -32,6 +32,18 @@ class TransactionHashPayload(BaseTxPayload):
     tx_hash: Optional[str]
 
 
+@dataclass(frozen=True)
+class SignaturePayload(BaseTxPayload):
+    """Represent a transaction payload of type 'signature'."""
+
+    signature: str
+
+
+@dataclass(frozen=True)
+class EmptyPayload(BaseTxPayload):
+    """Represent a transaction  empty payload."""
+
```

Comment:
Will become redundant if https://github.com/valory-xyz/price-oracle/pull/51/files#r1123092502 is taken into account.

Reply:
done, thanks